src/session/keys.ts

```diff
@@ -0,0 +1,20 @@
+export const SIGN_IN_INFO: string = "signin_info";
```

Comment:
A comment at the top of the file would be useful to explain what these are, where they come from etc.